R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
```isNull``` and ```isNotNull``` are still exported after this PR because they have been exist at Spark 1.5. ```isNull``` and ```isNotNull``` (with upper case) are functions of ```Column```, I consider  they are Spark specific functions so did not remove them. If we want to remove them, I can do it in a follow-up PR and they are breaking changes need to be explain in release note. @shivaram